File: sdks/swift/Sources/Schemas/UpdatePlanDtoBillingTiming.swift
import Foundation

/// When to charge: IN_ADVANCE or IN_ARREARS
public enum UpdatePlanDtoBillingTiming: String, Codable, Hashable, CaseIterable, Sendable {
    case inAdvance = "IN_ADVANCE"
    case inArrears = "IN_ARREARS"
}

File: sdks/ruby/lib/novabilling/plans/types/update_plan_dto_billing_timing.rb
# frozen_string_literal: true

module Novabilling
  module Plans
    module Types
      module UpdatePlanDtoBillingTiming
        extend Novabilling::Internal::Types::Enum

        IN_ADVANCE = "IN_ADVANCE"
        IN_ARREARS = "IN_ARREARS"
      end
    end
  end
end
